readme.md

```diff
@@ -10,9 +10,10 @@ It is built for use on http://savaslabs.com although you could easily adapt it f
 ### Installation
 
 ``` bash
-cd /path/to/squabble
```

Comment:
It would be good if we had `make` commands for local install. And we need to update all of these commands to run the container's php instead of the host.

We could use a similar process from other sites, where we pull in the seed DB via an AWS S3 cp command